frontend/src/components/Home.css

```diff
@@ -0,0 +1,161 @@
+      -3px 6px 1px hsl(0,0%,56%), 
+      -4px 8px 1px hsl(0,0%,58%), 
+      -5px 10px 1px hsl(0,0%,60%), 
+      -6px 12px 1px hsl(0,0%,62%);
+      margin-bottom: 20px;
+}
+.font-size-home-2{
+    font-size: 20px;
+    letter-spacing: .15em;
+    color: hsl(0,0%,94%);
+    text-shadow: 
+      -1px 2px 1px hsl(0,0%,52%), 
+      -2px 4px 1px hsl(0,0%,54%), 
+      -3px 6px 1px hsl(0,0%,56%), 
+      -4px 8px 1px hsl(0,0%,58%), 
+      -5px 10px 1px hsl(0,0%,60%), 
+      -6px 12px 1px hsl(0,0%,62%);
+      margin-bottom: 20px;
+      margin-top: 20px;
+      
+}
+
+.wrap-list-of-main-categories{
+    box-shadow: 0 2px 10px black;
+	border-radius: 5px;
+    height: 80%;
+    padding: 40px;
+}
+.wrap-list-of-main-categories-2{
+    box-shadow: 0 2px 10px black;
+	border-radius: 5px;
+    height: 80%;
+    padding: 40px;
+   
+}
+
+.list-of-main-categories{
+    list-style:none;
+    overflow: scroll;
+    width: 100%;
+    height: 90%;
+}
+
+.list-of-main-categories li a {
+    color: black;
+    text-decoration: none
+}
+.inside-wrap-div-3 a {
+    text-decoration: none
+}
+
+.div-box-slider-1{
+    box-shadow: 0 2px 10px black;
+	border-radius: 5px;
+    overflow: scroll;
+    padding: 5px;
+    height: 150px;
+    font-size: 14px;
+   
+}
+.categories-font{
+    text-shadow: 
+      -1px -1px 1px hsl(190,50%,90%), 
+      1px 1px 1px gray;
+      font-size: 20px;
+}
+
+.wrap-div-box-1{
+    padding-right: 40px;
+    padding-bottom: 20px;
+}
+
+
+.inside-wrap-div-1,.inside-wrap-div-2,.inside-wrap-div-3 {
+    
+    display: flex;
+    justify-content: space-between;
+}
+
+.Featured-content-slider,.Featured-category-1,.Featured-category-2,.Random-content {
+    padding-bottom: 20px;
+}
+.inside-wrap-div-2{
+    padding-left: 50px;
+    color: white;
+}
+.wellcom-div{
+    font-size: 40px;
+    text-align: center;
+    color: #A7D5D4;
+    text-shadow:
+      0 1px 0 hsl(185, 91%, 58%),
+      0 1px 0 hsl(185,91%,50%),
+      0 3px 0 hsl(185,91%,46%),
+      0 4px 0 hsl(185,90%,40%),
+      0 5px 0 hsl(185,90%,35%),
+      0 6px 1px hsl(185,90%,30%),
+      0 0 5px hsl(185,50%,25%),
+      0 1px 3px hsl(185,50%,25%),
+      0 3px 5px hsl(185,40%,25%),
+      0 5px 10px hsl(185,20%,25%),
+      0 10px 10px hsl(185,20%,25%);
+      padding-bottom: 20px;
+                
+}
+.inside-wrap-div-3{
+    padding-left: 20px;
+}
+@keyframes textshadow {
+	0%   { text-shadow: 1px 1px 0px #333 }
+	10%   { text-shadow: 3px 3px 2px #333 }
+	20%  { text-shadow: 9px 10px 6px #999 }
+	40%  { text-shadow: 10px 15px 6px #ccc }
+	60%  { text-shadow: 0px 10px 4px #ccc }
+	80%  { text-shadow: -10px 8px 3px #ccc }
+	100%  { text-shadow: 1px 5px 1px #eee }
+}
+.profiile{
+    width: 120px;
+    border: 2px solid black;
+    border-radius: 58px;
+    
+}
+.p-tag-profile{
+    text-shadow: 
+      -1px -1px 1px hsl(190,50%,90%), 
+      1px 1px 1px gray;
+      font-size: 25px;
+    color: white;
+    text-decoration: none;
+    
+    
+}
```

Comment:
Try to improve the code consistency a little. For example sometimes you have a space after the colon, like `color: white` and sometimes not, like `color:white`. It would be best to always have a space

Also sometimes you have an empty line after `}` and sometimes not, it would always be good to have an empty line after.

Also sometimes you have a space before `{` and sometimes not, it would be good to always have a space before it. 

These things appear small, but as the application grows they actually matter and quality code is consistent.